scripts/setup.sh

```diff
@@ -43,6 +43,11 @@ function maybeInstallDependencies {
       shift
       ;;
 
+      --suite=*)
```

Comment:
We are passing --suite parameter externally. Whereas --sharding is defined internally on run_e2e_tests.sh. When the run_e2e_tests.sh is executed, the parameters (--suite or --sharding or anything we pass externally) goes through setup.sh for dependencies installation. Here the error is raised, because the block 26 - 50 on setup.sh is designed to parse only --skip-install and --run-minified-tests.